src/test/java/duke/storage/StorageTest.java

```diff
@@ -0,0 +1,33 @@
+package duke.storage;
+
+import duke.exception.ChatBotParameterException;
+import duke.task.TaskList;
+import org.junit.jupiter.api.Test;
+
+import java.nio.file.Paths;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class StorageTest {
+
+    @Test
+    void createSaveDataFromTaskList_normalTask_success() throws ChatBotParameterException {
+        Storage storage = new Storage("test.txt");
+        String testCase = "E | 1 | hello | tmr | weekend\n" +
+                "D | 0 | deadline | 2002-09-05T16:00\n" +
+                "T | 0 | 1\n";
+        TaskList taskListToBeTested = new TaskList();
+        taskListToBeTested.addEvent("hello", "tmr", "weekend", true);
+        taskListToBeTested.addDeadline("deadline", "5/9/2002 1600", false);
+        taskListToBeTested.addToDo("1", false);
+        assertEquals(testCase, storage.createSaveDataFromTaskList(taskListToBeTested).toString());
+    }
+
+    @Test
+    void getFileLocationPath_normalFileName_success() {
```

Comment:
I liked that your test method name adheres to the coding standard, featureUnderTest_testScenario_expectedBehavior().

Reply:
Thanks!